code401Challenges/src/main/java/code401Challenges/RepeatedWord/RepeatedWord.java

```diff
@@ -0,0 +1,29 @@
+package code401Challenges.RepeatedWord;
+
+
+import java.util.HashMap;
+
+public class RepeatedWord {
+
+    public static String repeatedWord(String sentence){
+        String[] words = sentence.split(" ");
+        String temp = "";
+
+        if(words.length < 1){
+            throw new NullPointerException("empty string");
+        }
+
+        HashMap<String, Integer> hashMap = new HashMap<>();
+
+        for(String word : words){
+            //regex from stack overflow, see readme for link.
+            temp = word.toLowerCase().replaceAll("[^a-zA-Z]+$", "");
```

Comment:
Because you are using `.toLowerCase()` the `A-Z` in your regex isn't necessary (it doesn't hurt though)